src/main/java/org/jenkinsci/plugins/docker/workflow/WithContainerStep.java

```diff
@@ -177,6 +177,11 @@ private static void destroy(String container, Launcher launcher, Node node, EnvV
             }
 
             container = dockerClient.run(env, step.image, step.args, ws, volumes, volumesFromContainers, envReduced, dockerClient.whoAmI(), /* expected to hang until killed */ "cat");
+            final List<String> ps = dockerClient.listProcess(env, container);
```

Comment:
hesco on #jenkins reports that this line can itself fail, breaking the entire step for the sake of a diagnostic. Root cause unknown.